src/pudl/output/ferc1.py

```diff
@@ -3036,6 +3089,7 @@ def out_ferc1__yearly_rate_base(
         .assign(
             dollar_value=lambda x: x.dollar_value.divide(8),
             xbrl_factoid="cash_working_capital",  # newly definied (do we need to add it anywhere?)
+            tags_in_rate_base="yes",
```

Comment:
Can we get rid of this comment?

Reply:
i'll remove the () portion but i'd like to keep the newly defined bit